docs/cruisecontroloperation.md

```diff
@@ -0,0 +1,320 @@
+    id: 222e30f0-1e7a-4c87-901c-bed2854d69b7
+    operation: add_broker
+    parameters:
+      brokerid: "3"
+      exclude_recently_demoted_brokers: "true"
+      exclude_recently_removed_brokers: "true"
+    started: "2022-11-18T09:30:48Z"
+    state: Completed
+    summary:
+      Data to move: "0"
+      Intra broker data to move: "0"
+      Number of intra broker replica movements: "0"
+      Number of leader movements: "0"
+      Number of replica movements: "36"
+      Provision recommendation: '[ReplicaDistributionGoal] Remove at least 4 brokers.'
+      Recent windows: "1" 
+  errorPolicy: retry
+  retryCount: 0
+```
+
+The `status.currentTask` describes the Cruise Control task. The `httpRequest` field contains the whole POST HTTP request which has been executed. The `id` is the Cruise Control task identifier number. The `state` shows the progress of the add_broker request. The `summary` is the Cruise Control's optimization proposal. It shows the scope of the changes that Cruise Control will be applied through the operation. When a task execution completed with an error and there are re-executions the `retryCount` shows the number of the retries. In this case the history of the failed tasks are visible with the error message in the `status.failedTask` field list.  
+You can also find information from the fields [here](https://github.com/banzaicloud/koperator/blob/master/api/v1alpha1/cruisecontroloperation_types.go)
+
+## How can we have control over the created CruiseControlOperation?
+
+- The task execution can be stopped gracefully when the `CruiseControlOperation` is deleted. In this case the corresponding `cruiseControlState` or the `cruiseControlVolumeState` will get succeeded state.
+- `cruiseControlOperation.spec.errorPolicy` defines how the failed Cruise Control task should be handled. When the `errorPolicy` is **retry** (which is the default value), the Koperator re-executes the failed task in every 30 sec. When it is "ignore", the Koperator handles the failed task as completed thus the `cruiseControlState` or the `cruiseControlVolumeState` will get succeeded state.
+- When there is a Cruise Control task which can not be completed without an error and the  `cruiseControlOperation.spec.errorPolicy` is retry, the Koperator will re-execute the task until it is succeeded. This automatic re-execution can be paused with a label on the corresponding `CruiseControlOperation` CR (check the example below). When the **pause** label is not equal **true** or it the label is removed, the re-execution will be continued. This can be useful when the reason of the error cannot be fixed any time soon but you want to retry the operation later when the problem is solved. A paused `CruiseControlOperation` will not be considered when selecting operation for execution. It means when a new `CruiseControlOperation` with same operation (`status.currentTask.operation`) type is created it will be executed and the paused one will be skipped.
```

Comment:
```suggestion
- When there is a Cruise Control task which can not be completed without an error and the  `cruiseControlOperation.spec.errorPolicy` is retry, the Koperator will re-execute the task until it is succeeds. This automatic re-execution can be paused with a label on the corresponding `CruiseControlOperation` CR (check the example below). When the **pause** label does not equal **true** or if the label is removed, re-execution will continue. This can be useful when the reason of the error cannot be fixed any time soon but you want to retry the operation later when the problem is solved. A paused `CruiseControlOperation` will not be considered when selecting operation for execution. This means that when a new `CruiseControlOperation` with same operation (`status.currentTask.operation`) type is created it will be executed and the paused one will be skipped.
```